data/headerNavLinks.ts

```diff
@@ -4,6 +4,34 @@ const headerNavLinks = [
   { href: '/tags', title: 'Tags' },
   { href: '/projects', title: 'Projects' },
   { href: '/about', title: 'About' },
```

Comment:
We can type this as well: `const headerNavLinks: NavItem[]`

Reply:
Added type